Competition2018/src/main/java/competition/subsystems/wrist/commands/SetWristTo90Degrees.java

```diff
@@ -0,0 +1,37 @@
+package competition.subsystems.wrist.commands;
+
+import competition.subsystems.wrist.WristSubsystem;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class SetWristTo90Degrees extends SetWristAngleCommand{
```

Comment:
You don't need this command - your commandgroup can use the existing `SetWristAngleCommand`